src/storage/es_posix.c

```diff
@@ -192,6 +199,74 @@ es_rename_path (char *src, char *tgt, char *metaname)
 
   sprintf (t, "%s%s", metaname, s);
 }
+
+/*
+ * es_abs_open -
+ *
+ * return: return code of file open
+ * path(in): file path for open, relative or absolute
+ *           if relative, (lob_base_path) + path will be opened
+ */
+
+static int
+es_abs_open (const char *path, int flags)
+{
+  const char *abs_path = path;
+  char path_buf[PATH_MAX];
+  int ret = NO_ERROR;
+
+  if ((ret = es_make_abs_path (path_buf, path)) < 0)
+    {
+      return ret;
+    }
+
+  if (ret > 0)
+    {
+      abs_path = path_buf;
+    }
+
+  return open (abs_path, flags);
+}
+
+static int
+es_abs_open (const char *path, int flags, mode_t mode)
+{
+  const char *abs_path = path;
+  char path_buf[PATH_MAX];
+  int ret = NO_ERROR;
+
+  if ((ret = es_make_abs_path (path_buf, path)) < 0)
+    {
+      return ret;
+    }
+
+  if (ret > 0)
+    {
+      abs_path = path_buf;
+    }
+
+  return open (abs_path, flags, mode);
+}
+
+/*
+ * es_abs_open -
```

Comment:
```suggestion
 * es_make_abs_path -
```

Reply:
Thank you for your comment, its my mistake.